force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -126,18 +158,30 @@ public with sharing class ClassesObjectsMethods {
         List<String> scientistNames = new List<String>{'Tim Berners-Lee', 'Alan Turing', 'Grace Hopper', 'Donald Knuth', 'Guido van Rossum', 'Ken Thompson', 'Stephen Hawking'};
 
         // The variable to store the concatenated string
+        String result = '';
 
         // Loop through the list of scientists' names
-
+        for (String scientistName : scientistNames) {
             // If the name doesn't contain the letter 'G', skip this iteration
-
+            if (!scientistName.toLowerCase().contains('g')) {
+                continue;
+            }
             // Add the name to the result string, followed by a comma
+            result += scientistName + ',';
+        }
 
         // Remove the last comma from the result string
+        
+            result = result.removeEnd(',');
+        
 
-        return null; // Replace null with the variable you used to store the result
+        return result;
```

Comment:
No need for the continue block. If you check the positive condition 
```
   if (name.toLowerCase().contains('g'))
   ```
   Your loop variable automatically takes you to the next member if the condition is not met.